docs/conf.py

```diff
@@ -35,7 +35,7 @@
 import sphinx.environment
 from docutils.utils import get_source_line
 
-def _warn_node(self, msg, node):
+def _warn_node(self, msg, node, **kwargs):
     if not msg.startswith('nonlocal image URI found:'):
         self._warnfunc(msg, '%s:%s' % get_source_line(node))
 
```

Comment:
I don't think so. The doc is built correctly without. I added it because it is required now but I don't rely know what it is use for.